### PR TITLE
docs(README): refactor for webpack v2

### PR DESCRIPTION
**Pull request by michael-ciniawsky**
[README](https://github.com/michael-ciniawsky/webpack/tree/readme)

@webpack/core-team

### CODE REVIEW

**Comment by SpaceK33z** [comment]
Note that we should wait with merging this until webpack v2 is released.

File: README.md [hunk 1]
```diff
@@ -1,283 +1,487 @@
...
+	</a>
+	<a href="https://opencollective.com/webpack#sponsors">
+		<img src="https://opencollective.com/webpack/sponsors/badge.svg">
+	</a>
+	<a href="https://gitter.im/webpack/webpack">
+		<img src="https://badges.gitter.im/webpack/webpack.svg">
+	</a>
+  <h1>webpack</h1>
+  <p>
+    webpack is a bundler for modules. The main purpose is to bundle JavaScript files for usage in a browser, yet it is also capable of transforming, bundling, or packaging just about any resource or asset.
```
Comment (bebraw) [inline README.md]: Why `a bundler for modules` over `a module bundler`? Also maybe `Its main purpose is to bundle...` so it's more active.

File: README.md [hunk 2]
```diff
@@ -1,283 +1,487 @@
...
 
-**TL; DR**
-
-* Bundles both [CommonJS](http://wiki.commonjs.org/) and [AMD](https://github.com/amdjs/amdjs-api/wiki/AMD) modules (even combined).
+* Bundles [ES Modules](http://www.2ality.com/2014/09/es6-modules-final.html), [CommonJS](http://wiki.commonjs.org/) and [AMD](https://github.com/amdjs/amdjs-api/wiki/AMD) modules (even combined).
 * Can create a single bundle or multiple chunks that are asynchronously loaded at runtime (to reduce initial loading time).
-* Dependencies are resolved during compilation reducing the runtime size.
-* Loaders can preprocess files while compiling, e.g. coffeescript to JavaScript, handlebars strings to compiled functions, images to Base64, etc.
+* Dependencies are resolved during compilation, reducing the runtime size.
+* Loaders can preprocess files while compiling, e.g. CoffeeScript to JavaScript, Handlebars strings to compiled functions, images to Base64, etc.
```
Comment (bebraw) [inline README.md]: Is CoffeeScript relevant anymore? Maybe it would be nice to replace that with something like TypeScript?

File: README.md [hunk 3]
```diff
@@ -1,283 +1,487 @@
...
-project:
-`npm install webpack --save-dev`
-
-global:
-`npm install webpack -g`
+### [Get Started](https://webpack.js.org/get-started/)
 
-Usage:
-https://webpack.github.io/docs/tutorials/getting-started/
+Check out webpack's [docs](https://webpack.js.org/guides/) for quick Getting Started guide, in-depth usage, tutorials and resources.
```
Comment (bebraw) [inline README.md]: Why to link to the getting started above but not here? Maybe remove the link from the header and move it here.

File: README.md [hunk 4]
```diff
@@ -1,283 +1,487 @@
...
-* [`injectable`](https://github.com/jauco/webpack-injectable): Allow to inject dependencies into modules
-* [`transform`](https://github.com/webpack/transform-loader): Use browserify transforms as loader.
-* [`jsbeautify`](https://github.com/tomaszczechowski/jsbeautify-loader): Autoformatting code.
-
-For the full list of loaders, see [list of loaders](https://webpack.github.io/docs/list-of-loaders.html).
-
-## Module Format (AMD/CommonJS)
-
-webpack supports **both** AMD and CommonJS module styles. It performs clever static
+#### Files
```
Comment (bebraw) [inline README.md]: It might be a better idea to push these sort of lists to the documentation. Easier to maintain and that would slim down the readme as well.

File: README.md [hunk 5]
```diff
@@ -1,283 +1,487 @@
...
-	});
-}
-```
-
-## Documentation
-
-[documentation](https://webpack.github.io/docs/?utm_source=github&utm_medium=readme&utm_campaign=documentation)
+<h2 align="center">Example</h2>
 
+**app.js**
```
Comment (bebraw) [inline README.md]: Same here. Would it be better to defer to the docs to slim this down? Easier to maintain again.

File: README.md (hunk 4 above)
Comment (michael-ciniawsky) [inline README.md]: Yep agreed, since webpack.js.org recently got respective plugins && loaders lists, this is somehow redundant

File: README.md (hunk 5 above)
Comment (michael-ciniawsky) [inline README.md]: Maybe slimming it down to a very basic example like e.g on the docs splash page instead of removing it entirely?

File: README.md (hunk 2 above)
Comment (michael-ciniawsky) [inline README.md]: 👍

File: README.md (hunk 3 above)
Comment (michael-ciniawsky) [inline README.md]: hmm.. I'm not really happy with this section in general 😛. It should reflect the docs top-level structure somehow/better. I willl refactor that section when I something better in mind, but the various plugins && loaders docs first. Otherwise 👍 on the link change

**Comment by jhnns** [comment]
Maybe that should be a bigger change...

@sokra and I were thinking about removing all examples and user documentation from the repository. The repository should only contain contributor information.

**Comment by michael-ciniawsky** [comment]
@jhnns The main sections of webpack.js (Getting Started, Concepts, API) with a short overview and link to them should stay/ being added imo, agreed on removing the noisy loader && plugin lists and shorten or remove the unneeded example. The README should be a entry (index) to webpack.js.org short and precise + reflecting the community support

**Comment by jhnns** [comment]
@michael-ciniawsky that sounds good. I would also like to have all badges at one place :)

File: README.md (hunk 1 above)
Comment (SpaceK33z) [inline README.md]: fixed.

File: README.md (hunk 4 above)
Comment (SpaceK33z) [inline README.md]: I also agree, but lets keep them for now since we aren't done with setting it up on webpack.js.org yet.

File: README.md (hunk 3 above)
Comment (SpaceK33z) [inline README.md]: Fixed.

**Comment by SpaceK33z** [comment]
Fixed most of the feedback. The PR should be ready for merging after the CI is done.

Some things might still need a bit more fine tuning, but we can always do that later.
